modules/aws-workspace-basic/s3.tf

```diff
@@ -13,6 +13,13 @@ resource "aws_s3_bucket_versioning" "versioning" {
   }
 }
 
+resource "aws_s3_bucket_ownership_controls" "state" {
+  bucket = aws_s3_bucket.root_storage_bucket.id
+  rule {
+    object_ownership = "BucketOwnerPreferred"
+  }
+}
+
```

Comment:
Should we do the same for other AWS modules as well?

Reply:
Modules also affected with this issue, fixed here
- aws-exfiltration-protection
- aws-workspace-basic
- aws-workspace-with-firewall